### PR TITLE
Toolkit upgrade

### PR DESCRIPTION
**Pull request by mauricioac**

### CODE REVIEW

File: app/views/form_award_eligibilities/questions/_number_of_innovative_products.html.slim [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .question-body
-  = f.input :number_of_innovative_products, input_html: { class: 'small innovative_amount_input', autocomplete: "off" }, label: false
+  = f.input :number_of_innovative_products, input_html: { class: 'tiny innovative_amount_input', autocomplete: "off" }, label_html: { class: 'govuk-body govuk-!-font-weight-bold govuk-!-font-size-36 govuk-!-margin-bottom-5' }, label: @eligibility.class.label(step).html_safe
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/questions/_number_of_innovative_products.html.slim:2]: Can delete this file as well

File: app/views/form_award_eligibilities/questions/_qae_for_trade_award_year.html.slim [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.question-body
+  = f.input :qae_for_trade_award_year, as: :select, include_blank: true, label: false, input_html: { class: 'trade-awarded-input', autocomplete: "off" }
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/questions/_qae_for_trade_award_year.html.slim:2]: Can delete this file

File: app/views/form_award_eligibilities/questions/_sales_above_100_000_pounds.html.slim [hunk 3]
```diff
@@ -3,5 +3,5 @@
 
   - collection = @eligibility.class.sales_above_100_000_pounds.options
   - collection.reject! { |opts| opts.last == 'skip' } if controller_name == 'form_award_eligibilities'
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/questions/_sales_above_100_000_pounds.html.slim:5]: Can delete this file as well

File: app/views/form_award_eligibilities/questions/_sustainable_development.html.slim [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.question-body
+  - collection = @eligibility.class.sustainable_development.options
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/questions/_sustainable_development.html.slim:2]: Can delete this file

File: app/views/form_award_eligibilities/questions/hints/_promoting_social_mobility.html [hunk 5]
```diff
@@ -1,15 +1,15 @@
-<div class="question-context">
-  <p>Social mobility is a measure of the ability to move from a lower socio-economic background to higher socio-economic status.</p>
-  <ul>
+<div>
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/questions/hints/_promoting_social_mobility.html:1]: Can delete this file

File: app/views/form_award_eligibilities/questions/hints/_social_mobility_activities.html [hunk 6]
```diff
@@ -1,10 +1,10 @@
-<div class='question-context'>
-  <strong>Careers advice</strong> – providing careers advice or information to help people from disadvantaged backgrounds make more informed career choices as part of your recruitment initiatives such as traineeships, internships, apprenticeships or graduate schemes.<br/><br/>
-  <strong>Fairer recruitment</strong> – widening your recruitment pool and making your recruitment process fairer by assessing potential skills rather than background - socio-economic or academic.<br/><br/>
-  <strong>Skills development</strong> – providing activities or training to help people from disadvantaged backgrounds to develop hard skills (for example, numeracy, computer literacy, cooking) or soft skills (for example, workplace communication, effective workplace relationship development). This may include the development of aspirations and increasing motivation.<br/><br/>
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/questions/hints/_social_mobility_activities.html:4]: Can delete this file as well

File: app/views/form_award_eligibilities/show.html.slim [hunk 7]
```diff
@@ -1,68 +1,62 @@
...
           - if step && step.to_s != "wicked_finish"
             = simple_form_for @eligibility, as: :eligibility, url: wizard_path(step, form_id: @form_answer.id, skipped: params[:skipped]), method: :put do |f|
-              .current-question.qae-form#current-question
-                .question class="question-#{step.to_s.gsub!(/_/, '-')}"
-                  label.eligibilty-label for="eligibility_#{step.to_s}" id="eligibility_#{step.to_s}_label"
-                    h2
-                      = @eligibility.class.label(step).html_safe
+              .govuk-form-group.current-question.qae-form#current-question
+                fieldset.govuk-fieldset class="question-#{step.to_s.gsub!(/_/, '-')}"
+                  - if @eligibility.is_a?(Eligibility::Basic) || !input_steps.include?(step.to_s)
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/show.html.slim:14]: This condition is no longer needed

File: app/views/form_award_eligibilities/show.html.slim [hunk 8]
```diff
@@ -1,68 +1,62 @@
...
+                    legend.govuk-fieldset__legend.govuk-fieldset__legend--l.eligibilty-label for="eligibility_#{step.to_s}" id="eligibility_#{step.to_s}_label" class="govuk-!-margin-bottom-5"
+                      h2.govuk-fieldset__heading
+                        = @eligibility.class.label(step).html_safe
                   - if @eligibility.is_a?(Eligibility::Basic)
                     = render "form_award_eligibilities/basic_questions/#{step.to_s}", f: f, question: step.to_s
                   - else
```
Comment (dreamfall) [inline app/views/form_award_eligibilities/show.html.slim:20]: Actually this condition is no longer needed as well. That means you can remove `form_award_eligibilities/questions/` folder @mauricioac

File: app/views/qae_form/_by_trade_goods_and_services_label_question.html.slim [hunk 9]
```diff
@@ -0,0 +1,23 @@
+.js-by-trade-goods-and-services-amount role="group" id="q_#{question.key}" aria-labelledby="q_#{question.key}_label"
+  input name="#{question.input_name}[array]" value="true" type="hidden" *possible_read_only_ops
```
Comment (dreamfall) [inline app/views/qae_form/_by_trade_goods_and_services_label_question.html.slim:2]: Can delete this file

File: app/views/qae_form/_award_holder_question.html.slim [hunk 10]
```diff
@@ -10,7 +10,7 @@ div role="group" id="q_#{question.key}" aria-labelledby="q_#{question.key}_label
     - if question.entities.none?
       = render 'qae_form/award_fields', question: question, award: {}, index: 0
 
-a.button.button-add.js-button-add.if-no-js-hide href="#" data-entity="Award/Personal honour" *possible_read_only_ops
+a.govuk-button.govuk-button--secondary.button-add.js-button-add.if-no-js-hide href="#" data-entity="Award/Personal honour" *possible_read_only_ops
   | + Add another Award/Personal honour
 
-= link_to "+ Add Award/Personal honour", new_form_form_answer_awards_url(@form_answer.id, question.award_years_present ? {holder: true} : {}), class: "button button-add if-js-hide",  "data-entity" => "Award/Personal honour"
```
Comment (dreamfall) [inline app/views/qae_form/_award_holder_question.html.slim:16]: Can delete this file

File: app/views/qae_form/_by_years_label_question.html.slim [hunk 11]
```diff
@@ -3,31 +3,32 @@ div role="group" id="q_#{question.key}" aria-labelledby="q_#{question.key}_label
     - for c in question.by_year_conditions
       .js-conditional-question.js-fy-entry-container.by-years-wrapper data-question=question.step.form[c.question_key].parameterized_title data-value=c.question_value
         .if-js-hide
-          label
+          label.govuk-label
```
Comment (dreamfall) [inline app/views/qae_form/_by_years_label_question.html.slim:6]: Can delete this file

File: app/views/qae_form/_by_years_question.html.slim [hunk 12]
```diff
@@ -2,24 +2,23 @@
   - for c in question.by_year_conditions
     .js-conditional-question.by-years-wrapper data-question=question.step.form[c.question_key].parameterized_title data-value=c.question_value
       .if-js-hide
-        label
```
Comment (dreamfall) [inline app/views/qae_form/_by_years_question.html.slim:5]: Can delete this file

File: app/views/qae_form/_form_footer.html.slim [hunk 13]
```diff
@@ -15,29 +15,28 @@ footer
...
 
       - if current_form_is_editable?
         li.save-quit-link
           span.if-js-hide
-            - dashboard_link = current_subject.is_a?(Lieutenant) ? lieutenant_form_answer_url(@form_answer) : dashboard_url
-            = button_tag "Save and come back later", class: "save-quit-link btn", value: "redirect", name: "next_action", type: "submit", data: { url: dashboard_link }
+            = button_tag "Save and come back later", class: "govuk-button save-quit-link btn", value: "redirect", name: "next_action", type: "submit", data: { url: dashboard_url }
           span.if-no-js-hide
-            = link_to "Save and come back later", dashboard_link, class: "js-save-and-come-back"
+            = link_to "Save and come back later", dashboard_url, class: " govuk-button govuk-button--secondary js-save-and-come-back"
```
Comment (dreamfall) [inline app/views/qae_form/_form_footer.html.slim:28]: Need to bring back `dashboard_link` var.

File: app/views/qae_form/_form_footer.html.slim [hunk 14]
```diff
@@ -15,29 +15,28 @@ footer
...
 
-      - if step.next_for(@form_answer, current_form_user)
+      - if step.next
```
Comment (dreamfall) [inline app/views/qae_form/_form_footer.html.slim:30]: revert this change as well

File: app/views/qae_form/_one_option_by_years_question.html.slim [hunk 15]
```diff
@@ -1,24 +1,23 @@
 .js-financial-conditional data-first-year-min-value=question.first_year_min_value data-first-year-min-validation-message=question.first_year_validation_message role="group" id="q_#{question.key}" aria-labelledby="q_#{question.key}_label"
   .by-years-wrapper
     .if-js-hide
-      label
+      label.govuk-label
```
Comment (dreamfall) [inline app/views/qae_form/_one_option_by_years_question.html.slim:4]: Can delete this file

File: app/views/qae_form/_pdf_link.html.slim [hunk 16]
```diff
@@ -1,3 +1,6 @@
 - unless QAE.hide_pdf_links?
-  = link_to users_form_answer_path(@form_answer, format: :pdf) do
-    ' Download your nomination
+  = link_to users_form_answer_path(@form_answer, format: :pdf), class: 'govuk-link'
+    - if @form_answer.award_type == "promotion"
+      ' Download your nomination
```
Comment (dreamfall) [inline app/views/qae_form/_pdf_link.html.slim:4]: Should always be `nomination` for QAVS. Condition is no longer needed.

File: app/views/qae_form/_step_error.html.slim [hunk 17]
```diff
@@ -3,7 +3,17 @@
...
+          ' Your
+          - if @form_answer.award_type == "promotion"
+           ' nomination
+          - else
+            ' application
+          ' failed to submit because you've left some mandatory questions unanswered. These are flagged in red on the sidebar. Once they are all completed, you'll be able to submit your
+          - if @form_answer.award_type == "promotion"
+           ' nomination
+          - else
+            ' application
```
Comment (dreamfall) [inline app/views/qae_form/_step_error.html.slim:15]: Should always be nominstion.

File: app/views/qae_form/_steps_progress_bar.html.slim [hunk 18]
```diff
@@ -1,37 +1,37 @@
...
-                    span.step-number
-                      ' #{step_letters[step.index]}.
-                    = step.short_title.html_safe
-              - else
-                = link_to edit_form_path(@form_answer.id, step: step.title.parameterize), class: "js-step-link js-step-condition", data: { step: "step-#{step.title.parameterize}" } do
-                  span
-                    span.step-number
-                      ' #{step_letters[step.index]}.
-                    = step.short_title.html_safe
+          - for step in @form.steps do
```
Comment (dreamfall) [inline app/views/qae_form/_steps_progress_bar.html.slim:21]: Need to use `current_steps(@form_answer, current_form_user)` method

File: app/views/qae_form/_trade_commercial_success_question.html.slim [hunk 19]
```diff
@@ -0,0 +1,21 @@
+- condition = question.preselected_condition
+
```
Comment (dreamfall) [inline app/views/qae_form/_trade_commercial_success_question.html.slim:2]: Can delete this file

File: app/views/qae_form/confirm.html.slim [hunk 20]
```diff
@@ -5,57 +5,66 @@
...
     br
-    p
-      ' Your nomination was submitted. We have sent you an email confirming your submission.
+    p.govuk-body
+      ' Your nomination for The Queen's Award for Voluntary Service Award was submitted. We have sent you an email confirming your submission.
     br
-    p
+    p.govuk-body
+      strong
+        ' Your Unique Reference Number (URN) is #{@form_answer.urn}
```
Comment (dreamfall) [inline app/views/qae_form/confirm.html.slim:23]: This needs to be removed as we no longer have URN

File: app/views/qae_form/confirm.html.slim [hunk 21]
```diff
@@ -5,57 +5,66 @@
...
 
-        p
-          ' Return to
-          = link_to "Nominations", dashboard_path
-          '  or
-          - if current_admin
-            = link_to "Sign out", destroy_admin_session_path, method: :delete
-          - else
-            = link_to "Sign out", destroy_user_session_path, method: :delete
+
```
Comment (dreamfall) [inline app/views/qae_form/confirm.html.slim:70]: ✂️

File: app/views/qae_form/show.html.slim [hunk 22]
```diff
@@ -1,21 +1,28 @@
...
+          .govuk-error-summary__body
+            p.govuk-body
+              ' You need to request or upload at least 2 letters of support.
+            p.govuk-body
+              = link_to "Add more letters of support", form_form_answer_supporters_path(@form_answer), class: 'govuk-link'
+
+      = render partial: 'qae_form/step', collection: @form.steps, locals: {form: @form, answers: (HashWithIndifferentAccess.new(@form_answer.document || {})), attachments: HashWithIndifferentAccess.new(@attachments)}
 
-      = render "qae_form/step_error"
+      = render "qae_form/step_error"
```
Comment (dreamfall) [inline app/views/qae_form/show.html.slim:28]: ✂️

File: app/views/qae_form/show.html.slim [hunk 23]
```diff
@@ -1,21 +1,28 @@
...
+      - if @form_answer.validator_errors && @form_answer.validator_errors["supporters"].present?
+        .govuk-error-summary aria-labelledby="letters-of-support-error-title" role="alert" tabindex="-1" data-module="govuk-error-summary"
+          h2.govuk-error-summary__title#letters-of-support-error-title There was a problem submitting the form
+          .govuk-error-summary__body
+            p.govuk-body
+              ' You need to request or upload at least 2 letters of support.
+            p.govuk-body
+              = link_to "Add more letters of support", form_form_answer_supporters_path(@form_answer), class: 'govuk-link'
+
+      = render partial: 'qae_form/step', collection: @form.steps, locals: {form: @form, answers: (HashWithIndifferentAccess.new(@form_answer.document || {})), attachments: HashWithIndifferentAccess.new(@attachments)}
```
Comment (dreamfall) [inline app/views/qae_form/show.html.slim:26]: Should be `.current_steps(@form_answer, current_form_user)`

File: app/views/users/audit_certificate_mailer/notify.text.erb [hunk 24]
```diff
@@ -0,0 +1,9 @@
+<%= "Dear #{@recipient.full_name}," %>
+
```
Comment (dreamfall) [inline app/views/users/audit_certificate_mailer/notify.text.erb:2]: Can delete this file

File: app/views/users/audit_certificate_request_mailer/notify.text.erb [hunk 25]
```diff
@@ -0,0 +1,27 @@
+<%= "Dear #{@recipient.full_name}," %>
+
```
Comment (dreamfall) [inline app/views/users/audit_certificate_request_mailer/notify.text.erb:2]: Can delete this file

File: app/views/users/audit_certificate_request_mailer/preview/notify.html.slim [hunk 26]
```diff
@@ -0,0 +1,51 @@
+p style="font-size: 19px; line-height: 1.315789474; margin: 30px 0 30px 0;"
+  = "Dear #{@recipient.full_name},"
+
```
Comment (dreamfall) [inline app/views/users/audit_certificate_request_mailer/preview/notify.html.slim:3]: Can delete this file

File: app/views/users/audit_certificates/_file.html.slim [hunk 27]
```diff
@@ -0,0 +1,30 @@
+- if cert_exists
+  - att_record = award.audit_certificate
```
Comment (dreamfall) [inline app/views/users/audit_certificates/_file.html.slim:2]: Can delete this file

File: app/views/users/audit_certificates/_form.html.slim [hunk 28]
```diff
@@ -0,0 +1,19 @@
+= simple_form_for (audit_certificate || form_answer.build_audit_certificate),
+                  as: :audit_certificate,
```
Comment (dreamfall) [inline app/views/users/audit_certificates/_form.html.slim:2]: Can delete this file

File: app/views/users/audit_certificates/_non_js_form.html.slim [hunk 29]
```diff
@@ -0,0 +1,13 @@
+= simple_form_for (audit_certificate || form_answer.build_audit_certificate),
+                  as: :audit_certificate,
```
Comment (dreamfall) [inline app/views/users/audit_certificates/_non_js_form.html.slim:2]: Can delete this file

File: app/views/users/audit_certificates/_upload_block.html.slim [hunk 30]
```diff
@@ -0,0 +1,15 @@
+.js-upload-wrapper data-max-attachments=1 data-form-name="audit-cert-form" data-name="audit-cert-name" data-filename="Uploaded Verification of Commercial Figures"
+  ul.js-uploaded-list.no-bullets class=("hidden" unless cert_exists)
+    li.li-audit-upload
```
Comment (dreamfall) [inline app/views/users/audit_certificates/_upload_block.html.slim:3]: Can delete this file

File: app/views/users/audit_certificates/show.html.slim [hunk 31]
```diff
@@ -0,0 +1,32 @@
+- cert_exists = form_answer.audit_certificate.present? && form_answer.audit_certificate.persisted?
+- list_exists = form_answer.list_of_procedure.present? && form_answer.list_of_procedure.persisted?
+
```
Comment (dreamfall) [inline app/views/users/audit_certificates/show.html.slim:3]: Can delete this file

File: app/views/users/collaboration_mailer/access_granted.text.erb [hunk 32]
```diff
@@ -0,0 +1,27 @@
+Dear <%= @collaborator.full_name %>,
+
+<%= @user_who_added.full_name %> has invited you to collaborate with them on a Queen's Award for Enterprise application.
```
Comment (dreamfall) [inline app/views/users/collaboration_mailer/access_granted.text.erb:3]: Can delete this file

File: app/views/users/registrations/new.html.slim [hunk 33]
```diff
@@ -1,83 +1,64 @@
...
-
-          .form-inputs#email-control-group
-            .question
-              h2 = f.label :email
-              .question-body class="#{'no-error' if signup_email_already_exists?(f.object)}"
-                = f.input :email, required: true, autofocus: true, label: false, input_html: {autocomplete: "off", class: "large"}, error: !signup_email_already_exists?(f.object)
+      h1.govuk-heading-xl Create an account
+      .application-notice.info-notice
+        p.govuk-body
+          ' If you are a marketing company applying on behalf of a client, your client should create an account first, then add you as a collaborator.
```
Comment (dreamfall) [inline app/views/users/registrations/new.html.slim:9]: I don't think this copy is correct

File: app/views/users/supporter_mailer/success.text.erb [hunk 34]
```diff
@@ -0,0 +1,18 @@
+<%= "Dear #{@supporter.full_name}," %>
+
+You've been chosen by <%= @nominator_name %> to write a letter of support in aid of the nomination of <%= @nominee_name %> for a Queen's Award for Enterprise Promotion.
```
Comment (dreamfall) [inline app/views/users/supporter_mailer/success.text.erb:3]: Can delete this file

**Review by dreamfall — CHANGES_REQUESTED** [review]

File: app/views/content_only/_award_nickname.html.slim [hunk 35]
```diff
@@ -1,25 +1,38 @@
...
+      ul.govuk-list.govuk-list--bullet
+        li
+          ' The deadline for submissions is
+          = submission_deadline.decorate.formatted_trigger_time
+          ' .
+        li
+          ' You can still edit submitted nominations up to this date.
+
+  - if award_type == "innovation"
+    = render "content_only/award_nickname_block"
```
[the remark this answers is not in the document]
Comment (dreamfall) [reply app/views/content_only/_award_nickname.html.slim]: @mauricioac I think you've missed it

```
  - if award_type == "innovation"
    = render "content_only/award_nickname_block"
```

This is no longer needed.

File: app/views/qae_form/_form_footer.html.slim (hunk 13 above)
Comment (dreamfall) [reply app/views/qae_form/_form_footer.html.slim:28]: https://github.com/bitzesty/qavs-v2/pull/51/files#diff-d8ae559f0c087f6feb9cdd0a046fa2eae0f57a86e5d44283f845602ea44b5314L26

@mauricioac here's the link to var (it was removed as well)

File: config/database.yml [hunk 36]
```diff
@@ -1,6 +1,8 @@
 default: &default
   adapter: postgresql
   encoding: utf8
+  username: postgres
+  password: postgres
```
Comment (dreamfall) [inline config/database.yml:5]: Need to probably check out this file 👮

**Review by dreamfall — CHANGES_REQUESTED** [review]

File: config/database.yml (hunk 36 above)
Comment (matthewford) [reply config/database.yml:5]: I'm going to be changing these soon anyway in my pr to use env vars.

**Pull request merged**